contracts/hardhat.config.ts

```diff
@@ -60,15 +78,16 @@ const config: HardhatUserConfig = {
     timeout: 200000, // 200 seconds max for running tests
   },
   etherscan: {
-    apiKey: {
-      polygonMumbai: POLYGONSCAN_KEY,
-    },
+    // apiKey: {
+    //   polygonMumbai: POLYGONSCAN_KEY,
+    // },
+    apiKey: ETHERSCAN_API_KEY,
   },
 };
 
 export default config;
 
-// npx hardhat deploy --network mumbai --new-proxy-instance --name "FleekNFAs" --symbol "FLKNFA" --billing "[10000, 20000]"
+// npx hardhat deploy --network mumbai/sepolia/goerli --new-proxy-instance --name "FleekNFAs" --symbol "FLKNFA" --billing "[10000, 20000]"
```

Comment:
I don't think that would be the best approach because it can raise confusion as to why just one network is mentioned there. I will change it to `[mumbai | sepolia | goerli]`. Please let me know if you think that's good or if you still think adding another comment is the better option.

Reply:
I don't think that approach is good either because then you can't copy and paste this comment. What is the purpose of putting this comment here in the first place? I think what I originally proposes is better. I.e.:

```
// Use the following command to deploy where the network flag can be replaced with the network you choose:
// npx hardhat deploy --network goerli --new-proxy-instance --name "FleekNFAs" --symbol "FLKNFA" --billing "[10000, 20000]"
```